web/src/components/FeedbackModal.tsx

```diff
@@ -69,30 +72,42 @@ type FeedbackModalProps = {
   cityCode: string
   language: string
   routeType: RouteType
-  feedbackRating: FeedbackRatingType
+  visible: boolean
   closeModal: () => void
 }
 
 const FeedbackModal = (props: FeedbackModalProps): ReactElement => {
-  const { feedbackRating, closeModal, ...otherProps } = props
+  const { visible, closeModal, ...otherProps } = props
+  const [isSubmitted, setIsSubmitted] = useState<boolean>(false)
   const { t } = useTranslation('feedback')
+  const headline = isSubmitted ? `${t('thanksHeadline')}` : `${t('headline')}`
+  const { viewportSmall } = useWindowDimensions()
+  useScrollToTop()
+  useLockedBody(visible)
 
   return (
     <FocusTrap focusTrapOptions={{ clickOutsideDeactivates: true }}>
-      <ModalContainer role='dialog' aria-modal>
+      <ModalContainer
+        role='dialog'
+        aria-modal
+        topPosition={viewportSmall ? dimensions.headerHeightSmall : 0}
```

Comment:
Shouldn't we really do it fullscreen and not show the header? I think otherwise it doesn't really look like a modal/jumps a bit.

Reply:
i prefer showing the header on mobile as shown in design